core/src/main/java/org/apache/accumulo/core/file/rfile/PrintInfo.java

```diff
@@ -119,6 +182,18 @@ public String keyword() {
     return "rfile-info";
   }
 
+  protected Class<? extends Formatter> getFormater(String formatterClazz) {
+    try {
+      if (formatterClazz != null) {
+        return this.getClass().getClassLoader().loadClass(formatterClazz)
+            .asSubclass(Formatter.class);
+      }
+    } catch (ClassNotFoundException e) {
+      System.err.println("Could not find formatter class: " + formatterClazz);
+    }
+    return null;
+  }
+
```

Comment:
This is a general question - would this reflection code need to change for jdk-11+ (for when this is merged forward)? Specifically, is this still the standard way to use reflection, or has the "standard" changed?  From what I've read it looks okay, but thought it warranted checking.

Reply:
I think this will work fine in Java 11, since it uses the class loader of the current class, and not the system class loader.